test/jdk/com/sun/jdi/JdbLastErrorTest.java

```diff
@@ -27,7 +27,7 @@
  * @summary Test persistence of native last error value under jdb (Windows)
  * @requires (os.family == "windows")
  * @library /test/lib
- * @run compile --release 20 --enable-preview JdbLastErrorTest.java
+ * @run compile --release ${jdk.version} --enable-preview JdbLastErrorTest.java
  * @run main/othervm --enable-preview JdbLastErrorTest
```

Comment:
Better still would be to add  `@enablePreview`. That would allow you the `@run compile` line to go away and you can drop the `--enable-preview` from the `@run main`.

Reply:
Is that a new jtreg feature? I've used the idiom in the updated version of this file elsewhere in the JDK.

Reply:
https://bugs.openjdk.org/browse/CODETOOLS-7902654
I only became aware of it a few months ago.

Reply:
Thanks @AlanBateman; I wasn't aware of the feature. Looks like there is still usage of the old idiom.